### PR TITLE
Upgrading to latest syn and deku versions

### PR DESCRIPTION
**Pull request by karnotxo**
Just changed versions and fixed compilation issues

### CODE REVIEW

**Comment by wcampbell0x2a** [comment]
Thanks, could you remove the `.gitignore`, `README.md`, and `.idea` changes?

File: asterix-derive/src/lib.rs [hunk 1]
```diff
@@ -117,6 +115,6 @@ pub fn derive_answer_fn(input: TokenStream) -> TokenStream {
             }
         }
     };
-
-    expanded.into()
+    // Hand the output tokens back to the compiler
+    TokenStream::from(expanded) // als  could be 'expanded.into()'
```
Comment (wcampbell0x2a) [inline asterix-derive/src/lib.rs]:
```suggestion
    TokenStream::from(expanded) // also could be 'expanded.into()'
```

**Comment by wcampbell0x2a** [comment]
Ignore the failing CI, that's not related to your changes directly (except for the code coverage, but it's fine for now)

**Comment by karnotxo** [comment]
> Thanks, could you remove the `.gitignore`, `README.md`, and `.idea` changes?

Sorry... Still don´t figure how to do it... Will try.

**Comment by karnotxo** [comment]
> Thanks, could you remove the `.gitignore`, `README.md`, and `.idea` changes?

I think it's done. Plz, review. Thanks

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ jobs:
         toolchain:
           - stable
             # msrv
-          - 1.56.1
```
Comment (wcampbell0x2a) [inline .github/workflows/main.yml:23]: `cargo msrv` is reporting `1.66.1` as the msrv. I'd move this number to that one.

**Pull request merged**